<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbDDA">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Цифровой дифф. анализатор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbBrezenhemFloat">
        <property name="text">
         <string>Брезенхем (float)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbBrezenhemInt">
        <property name="text">
         <string>Брезенхем (int)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbBrezenhemAntiLader">
        <property name="text">
         <string>Брезенхем с устранением ступенчатости</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbWu">
        <property name="text">
         <string>Ву</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbStandart">
        <property name="text">
         <string>Стандартный</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Начало отрезка</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QSpinBox" name="x1SpinBox">
                <property name="suffix">
                 <string/>
                </property>
                <property name="prefix">
                 <string>X1: </string>
                </property>
                <property name="minimum">
                 <number>-10000</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="y1SpinBox">
                <property name="prefix">
                 <string>Y1: </string>
                </property>
                <property name="minimum">
                 <number>-10000</number>
                </property>
                <property name="maximum">
                 <number>10000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Конец отрезка</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QSpinBox" name="x2SpinBox">
              <property name="prefix">
               <string>X2: </string>
              </property>
              <property name="minimum">
               <number>-10000</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>350</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="y2SpinBox">
              <property name="prefix">
               <string>Y2: </string>
              </property>
              <property name="minimum">
               <number>-10000</number>
              </property>
              <property name="maximum">
               <number>10000</number>
              </property>
              <property name="value">
               <number>350</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="angleSpinBox">
          <property name="prefix">
           <string>Угол поворота: </string>
          </property>
          <property name="minimum">
           <number>-360</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>15</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lineSpinBox">
          <property name="prefix">
           <string>Длина линии: </string>
          </property>
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="colorLineButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Выбрать цвет линии</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fgLabel">
          <property name="text">
           <string>Цвет линии</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="colorBackButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Выбрать цвет фона (Нет)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Очитстить экран</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="spectrDrawButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Построить Спектр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lineDrawButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Построить линию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="calcTimeButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>Сравнить время построеия</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compareLadderButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>9</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Сравнить ступенчатость</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>2</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
